lib/vizkit/connection_manager.rb

```diff
@@ -48,17 +44,23 @@ def listening?(port = nil)
             end
         end
 
-        def disconnect(port=nil)
+        def disconnect(port=nil, keep_port=true)
```

Comment:
Please use a keyword argument here

```
def disconnect(port = nil, keep_port: true)
end
```

and `disconnect(port, keep_port: false)`